Userland/Libraries/LibVideo/VP9/Utilities.cpp

```diff
@@ -8,14 +8,4 @@
 
 namespace Video::VP9 {
 
-u8 clip_3(u8 x, u8 y, u8 z)
```

Comment:
In commit `LibVideo: Implement VP9 intra-predicted frame decoding` Utilities.cpp becomes completely empty. Do more utilities find their way here later? if not, the file should be deleted in that commit.

Reply:
My fault, I had meant to delete the file when I committed my changes. Fixed.